lib/plugins/index.js

```diff
@@ -1,12 +1,14 @@
 const { pluginPingPong } = require('./ping-pong')
 const { pluginWelcomeMessage } = require('./welcome-message')
 const { pluginRandomMessages } = require('./random-messages')
+const { pluginChuckNorrisJokes } = require('./chuck-norris')
```

Comment:
Kérlek vidd át egy "chuck-norris-jokes" mappába inkább. Hogy egységes legyen.
```suggestion
const { pluginChuckNorrisJokes } = require('./chuck-norris-jokes')
```

Reply:
jogos, egyértelműbb ha könyvtár névben is következetesek vagyunk